src/tests/functional/plugin/conformance/subgraphs_dumper/tests/matchers/generic_single_op.cpp

```diff
@@ -5,7 +5,7 @@
 #include "gtest/gtest.h"
 #include "matchers/single_op.hpp"
 #include "ngraph/ops.hpp"
-#include "functional_test_utils/include/functional_test_utils/summary/op_info.hpp"
+#include "functional_test_utils/summary/op_info.hpp"
```

Comment:
This target will be removed soon. No need to update

Reply:
CI fails if I don't update it :)